modules/javafx.controls/src/main/java/javafx/scene/control/Skinnable.java

```diff
@@ -36,29 +36,22 @@
  */
 public interface Skinnable {
     /**
-     * Skin is responsible for rendering this {@code Control}. From the
-     * perspective of the {@code Control}, the {@code Skin} is a black box.
-     * It listens and responds to changes in state in a {@code Control}.
+     * The Skin is responsible for rendering this {@code Skinnable}. From the
+     * perspective of the {@code Skinnable}, the {@code Skin} is a black box.
+     * It listens and responds to changes in state in a {@code Skinnable}.
```

Comment:
I think the start is incorrect (in both old and new version).

It should either be "The skin **that** is responsible for" or "The skin responsible for".

```suggestion
     * The Skin responsible for rendering this {@code Skinnable}. From the
     * perspective of the {@code Skinnable}, the {@code Skin} is a black box.
     * It listens and responds to changes in state of its {@code Skinnable}.
```